src/user_management/user.resolver.js

```diff
@@ -12,20 +16,25 @@ export default {
     }
   },
   Query: {
-      logInUser: async (email, password) => {
-        const user = await models.user.findOne({ where: { email } })
-        
-        if (!await user?.validatePassword(password, user.password)) {
-          throw new Error('Invalid email or password')
-        }
-        
-        return user
-      },
-      users: () => {
-        return models.user.findAll()
-      },
-      currentUser: () => {
-        return models.user.findOne()
+    logInUser: async (email, password) => {
+      const user = await models.user.findOne({ where: { email } })
+      if (!await user?.validatePassword(password, user.password)) {
+        throw new Error('Invalid email or password')
+      }
+      const accessToken = sign({ userId: user.id }, 'secret', { algorithm: 'RS256' }, { expiresIn: '15m' })
+      setAccessToken(accessToken)
+      const result = getDecodedAccessToken()
+      console.log(result)
+      return {
+        accessToken,
+        user
       }
```

Comment:
- Don't upload code with `console.log`s used for debugging.
- There's no need to return the `accessToken`.